src/App.vue

```diff
@@ -35,13 +35,22 @@ export default {
   created () {
     this.$store.dispatch('bindTopics')
     this.$store.dispatch('onAuthStateChanged')
+    this.setLanguage()
   },
   mixins: [notification, fb],
   computed: {
     ...mapGetters({
       isLoggedIn: 'isLoggedIn',
       isAdmin: 'isAdmin'
     })
+  },
+  methods: {
+    setLanguage () {
+      const lang = localStorage.getItem('lang')
```

Comment:
It will be nice to add

```js
if (typeof localStorage === 'undefined') {
  return
}
```

because there is quite a lot of people with blocked any kind of storage options

Reply:
added in `App` and `LangSwitcher`